tests/PHPUnit/Unit/Period/RangeTest.php

```diff
@@ -302,7 +302,7 @@ public function testRangeYearcomma1()
                 9  => '2007-10-01',
                 10 => '2007-11-01',
                 11 => '2007-12-01',
-            )),
+                )),
```

Comment:
We should indent the whole array above here. Not sure why we have the 1 as an explicit key there, it would be implied in a numerically indexed array (I'd think).